lib/models/entitiesStore.js

```diff
@@ -28,15 +28,89 @@ var async = require('async'),
     entitiesCollectionName = require('../../config').orionDb.collection,
     myutils = require('../myutils'),
     constants = require('../constants'),
-    logger = require('logops');
+    logger = require('logops'),
+    NGSI = require('ngsijs');
```

Comment:
it's a reference identifier. It should be lowercase. To be NGSI it's not enougth :)

Reply:
Solved in this commit:
https://github.com/telefonicaid/perseo-fe/pull/726/commits/184f6b7a3759d36dc5ee07f176925d9fdc049b42